background.html

```diff
@@ -32,12 +32,15 @@
       {{ /hideProgress }}
       <div class='message'>{{& message }}</div>
       <div>
-        {{ #exportButton }}
-          <button class='export grey'>{{ exportButton }}</button>
-        {{ /exportButton }}
-        {{ #debugLogButton }}
-          <button class='debug-log grey'>{{ debugLogButton }}</button>
-        {{ /debugLogButton }}
+        {{ #importButton }}
```

Comment:
Ah, that standard problem of pithy UI text in a world of infinite contexts... :0) Will think about this.